.travis.yml

```diff
@@ -42,6 +43,15 @@ matrix:
           os: osx
           osx_image: xcode8.3
           env: TOXENV=py37
+    allow_failures:
+        - language: generic
+          os: osx
+          osx_image: xcode8.3
+          env: TOXENV=py35
+        - language: generic
+          os: osx
+          osx_image: xcode8.3
+          env: TOXENV=py37
```

Comment:
isn't this duplicating the entries we already have above?

Reply:
Yes, that's how I remembered it. See 9c4bbdd1329e866f5517e35150ea02501e87de37 where I removed the entries from above, then Travis CI doesn't do the builds from `allow_failures` at all: https://travis-ci.org/borgbackup/borg/builds/653877191

However, I agree, it's not nice to have the osx entries both under `include` as well as under `allow_failures`, so with a44c545278000d3b3ba3ac72b315fde2f95f17a7 I just universally marked all builds with `os: osx` as allowed failures. Now the jobs run as expected again: https://travis-ci.org/borgbackup/borg/builds/653879285 And since `os: osx` is also the reason why the builds even fail in the first place, I think universally marking `os: osx` should be fine, too. (At least for the time being.)